backend/coins/btc/account_test.go

```diff
@@ -193,3 +184,17 @@ func TestInsuredAccountAddresses(t *testing.T) {
 	require.Equal(t, *account2.GetUnusedReceiveAddresses()[0].ScriptType, signing.ScriptTypeP2WPKH)
 
 }
+
+func TestSignAddress(t *testing.T) {
+	account := mockAccount(t, nil)
+	require.NoError(t, account.Initialize())
+	// pt2r is not an available script type in the mocked account.
+	_, _, err := btc.SignBTCAddress(account, "Hello there", "p2tr")
+	require.NotNil(t, err)
+	address, signature, err := btc.SignBTCAddress(account, "Hello there", "p2wpkh")
+	require.Nil(t, err)
+	require.NotNil(t, address)
```

Comment:
address is a string, strange to compare for `nil`. probably `require.NotEmpty()` is better.